lib/block_view_stake.go

```diff
@@ -1476,8 +1481,9 @@ func (bav *UtxoView) _connectUnstake(
 		Type:                       OperationTypeUnstake,
 		PrevValidatorEntry:         prevValidatorEntry,
 		PrevGlobalStakeAmountNanos: prevGlobalStakeAmountNanos,
-		PrevStakeEntries:           []*StakeEntry{prevStakeEntry},
-		PrevLockedStakeEntries:     []*LockedStakeEntry{prevLockedStakeEntry},
+		PrevStakeEntries:           prevStakeEntries,
+		PrevLockedStakeEntries:     prevLockedStakeEntries,
+		PrevEpochNumber:            currentEpochNumber,
```

Comment:
Why do we need `PrevEpochNumber` in the utxo op? Why can't we look it up in disconnect? jw

Reply:
I think you're right. If we ever disconnect a transaction, it'll be in the same block/epoch as the connect logic was ran. I removed.